python/sbp/client/framer.py

```diff
@@ -101,29 +104,33 @@ def _receive(self):
     # empty input
     preamble = self._read(1)
```

Comment:
I think one potential performance improvement is to read more than 1 byte or one payload at a time from the serial port.  It might be better to read whatever is there with every read call and have the framer use a software buffer to deframe into messages.